openforcefield/typing/engines/smirnoff/forcefield.py

```diff
@@ -245,6 +247,9 @@ def __init__(self,
             This option is primarily intended for forcefield development.
         allow_cosmetic_attributes : bool, optional. Default = False
             Whether to retain non-spec kwargs from data sources.
+        load_plugins: bool, optional. Default = True
+            Whether to load ``ParameterHandler`` classes which have been registered
+            by installed plugins.
```

Comment:
This will be the only mention of this entrypoint in our FF docs, so could you mention the specific path (`openff.toolkit.plugins.handlers`) here?